CONTRIBUTING.md

```diff
@@ -39,12 +39,14 @@ under the LICENSE file in the root directory of this source tree.
 
 ## Contributing Guidelines
 
+Note: To accelerate contributions to and innovations around `torchtitan`, we are adding a new, experimental folder [`torchtitan/experiments`](torchtitan/experiments/), which has its own [Contributing Guidelines](torchtitan/experiments/README.md#contributing-guidelines). The content below is for the rest, core part of `torchtitan`.
+
 ### Principles of contribution
 
 - Apply PyTorch-native training techniques.
   - The technique should be of general interests for distributed training.
   - A technique with moderate to large complexity should be sitting in the proper repo (e.g. pytorch/pytorch for a new parallelism, or pytorch/data for a new data loader) instead of `torchtitan`.
-  - The main branch of `torchtitan` should have minimal dependency on non-PyTorch libraries. Interesting models/techniques that depend on external libraries can be demonstrated in forks of `torchtitan`.
+  - The main branch of `torchtitan` should have minimal dependency on non-PyTorch libraries. Interesting models/techniques that depend on external libraries can be demonstrated in the `experiments` folder, or in forks of `torchtitan`.
 - Aim for minimum (if not zero) code change to the model. For the Llama model in `torchtitan`, if one has to make (justifiable) model change:
```

Comment:
- Aim for minimal 
(rather than minimum) 
... if one has to make justifiable model change(s): 
(easier to read by dropping the parens for justifiable, added (s) for changes since it could one or more justifiable changes)